python/nav/web/auth.py

```diff
@@ -78,3 +94,39 @@ def authenticate(username, password):
         return account
     else:
         return None
+
+
+def authenticate_remote_user(request=None):
+    if not NAV_CONFIG.get('AUTH_SUPPORT_REMOTE_USER', False):
+        return None
+
+    if not request:
+        return None
+
+    username = request.META.get('REMOTE_USER', None)
```

Comment:
Perhaps include a call to `.strip()` here, to protect against an authenticator that provides faulty input data?

Reply:
Fixed in "Protect against whitespace-only username"